homeassistant/components/webhook/__init__.py

```diff
@@ -71,7 +71,17 @@ async def async_handle_webhook(hass, webhook_id, request):
 
     # Always respond successfully to not give away if a hook exists or not.
     if webhook is None:
-        _LOGGER.warning("Received message for unregistered webhook %s", webhook_id)
+        host = STATE_UNKNOWN
+        peername = request.transport.get_extra_info("peername")
+        if peername is not None:
+            host, _ = peername
+        _LOGGER.warning(
+            "Received message for unregistered webhook %s from %s", webhook_id, host
+        )
+        # Look at content to provide some context for received webhook
+        content = await request.content.read()
```

Comment:
Can we just read the first 64 bytes, instead of reading it all into memory ?

Reply:
Good point. Done.